en/docs/learn/configuring-email-otp.md

```diff
@@ -31,52 +31,40 @@ configure Gmail or SendGrid as the email OTP provider using Gmail or
 SendGrid APIs. Follow the instructions in **one** of these sections to
 set up the email OTP provider.
 
-#### Configure WSO2 IS as the email OTP provider
+#### Option1: Configure WSO2 IS as the email OTP provider
 
 Follow the steps below to configure WSO2 IS to send emails once the
 Email OTP is enabled.
 
 1.  Shut down the server if it is running.
-2.  Open the
-    `           <IS_HOME>/repository/conf/axis2/axis2.xml          `
-    file, uncomment the
-    `                       transportSender                                  name                                  =                     `
-    "mailto" configurations, and update the following properties:
+2.  Add the following properties to the `deployment.toml` file in the `IS_HOME/repository/conf` folder to configure the email server.
 
+    ```
+           [output_adapter.email]
+           from_address= "wso2iamtest@gmail.com"
+           username= "wso2iamtest"
+           password= "Wso2@iam70"
+           hostname= smtp.gmail.com
+           port= 587
+           enable_start_tls= true
+           enable_authentication= true
```

Comment:
Fix indentation (see sample below):

```
form_address= "wso2iamtest@gmail.com"
```

Try to fix in other applicable places as well.